src/ui/components/atoms/label-date/LabelDate.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import type { StoryObj, Meta } from '@storybook/react';
+import LabelDate from './LabelDate';
+
+const meta: Meta<typeof LabelDate> = {
+  title: 'ui/components/atoms/label-date',
+  component: LabelDate,
+  argTypes: {},
+};
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+const createStory = (month: string, day: number, year: number): Story => ({
+  render: (args) => <LabelDate month={args.month} day={args.day} year={args.year} />,
+  args: {
+    month,
+    day,
+    year,
+  },
+});
+
+export const LabelDate1 = createStory('January', 1, 2005);
```

Comment:
I don't agree with this approach, I think is better to pass a Date object directly to this component, and set up a transformer or mapping date inside, which makes more easier to manage.
And why not, you can add another prop like an overflow parameters, with the month, day and year